lib/iiif_print.rb

```diff
@@ -1,12 +1,12 @@
-require "iiif_print/engine"
-require "iiif_print/errors"
-require "iiif_print/jp2_image_metadata"
-require "iiif_print/image_tool"
-require "iiif_print/issue_pdf_composer"
-require "iiif_print/text_extraction"
-require "iiif_print/data"
-require "iiif_print/configuration"
-require "iiif_print/resource_fetcher"
+require "iiif_print_config/engine"
```

Comment:
Were these changes a result of an over-aggressive rename?

Reply:
Yeah, those are totally not something that should be there.  Revisiting